codeworld-api/src/CodeWorld/Picture.hs

```diff
@@ -197,24 +206,21 @@ solidLoop :: HasCallStack => [Point] -> Picture
 solidLoop ps = SolidClosedCurve callStack ps
 
 {-# WARNING solidLoop "Please use solidClosedCurve instead of solidLoop." #-}
+rectangleToPolygon :: CallStack -> Double -> Double -> Picture
```

Comment:
Did you mean to group this together with the WARNING pragma?  Why?

Reply:
didn't mean to. Removed it.